internal/k8shandler/service.go

```diff
@@ -40,3 +43,13 @@ func (clusterRequest *ClusterLoggingRequest) RemoveInputService(serviceName stri
 
 	return nil
 }
+
+// GetServiceList returns a list of services based on a key/value label
+func (clusterRequest *ClusterLoggingRequest) GetServiceList(key string, val string) (*core.ServiceList, error) {
```

Comment:
Since we're only going to be looking at the services owned by a specific CLF, would it make sense to query here only for services in the same namespace as the CLF?

Reply:
Yes that is a good idea so as to reduce the number of services listed. I will make the change.